src/components/shapediver/parameter/ParameterFileInputComponent.tsx

```diff
@@ -1,12 +1,37 @@
 import { FileInput } from "@mantine/core";
 import { IconUpload } from "@tabler/icons-react";
 import React, { useEffect } from "react";
-import { extendMimeTypes, mapMimeTypeToFileEndings } from "@shapediver/viewer.utils.mime-type";
+import { extendMimeTypes, guessMimeTypeFromFilename, mapMimeTypeToFileEndings } from "@shapediver/viewer.utils.mime-type";
 import ParameterLabelComponent from "components/shapediver/parameter/ParameterLabelComponent";
 import { PropsParameter } from "types/components/shapediver/propsParameter";
 import { useParameterComponentCommons } from "hooks/shapediver/parameters/useParameterComponentCommons";
 import { isFileParameter } from "types/shapediver/viewer";
 
+/**
+ * Guess the mime type of a file by its extension.
+ * @param fileName 
+ * @returns 
+ */
+const guessMimeTypeByExt = (fileName: string) => {
+	return guessMimeTypeFromFilename(fileName)[0];
+};
+
+/**
+ * In case a file is missing a mime type, try to guess it from the file name.
+ * @param file 
+ * @returns 
+ */
+const guessMissingMimeType = (file: File | string) : File | string => {
+	if (typeof(file) === "string") {
+		return file;
+	}
+	if (file.type) {
+		return file;
+	}
+
+	return new File([file], file.name, { type: guessMimeTypeByExt(file.name) });
+};
+
```

Comment:
I plugged the logic we use on the frontend. It would be better to do this in the implementation of the file parameter in the viewer code. @MajorMeerkatThe3rd